driver/src/main/java/oracle/nosql/driver/ops/serde/nson/NsonProtocol.java

```diff
@@ -86,6 +86,7 @@ public class NsonProtocol {
     public static String START = "sr";
     public static String STATEMENT = "st";
     public static String STORAGE_THROTTLE_COUNT = "sl";
+    public static String SYSTEM = "sy";
```

Comment:
Isn't this supposed to be "sm" ?

Reply:
The "sm" has been used by another field [SERVER_MEMORY_CONSUMPTION](https://github.com/oracle/nosql-java-sdk/blob/main/driver/src/main/java/oracle/nosql/driver/ops/serde/nson/NsonProtocol.java#L86C5-L86C59), the SERVER_MEMORY_CONSUMPTION is not included into NsonProtocol of proxy cloud-5.12, so I missed it.

```
public static String SERVER_MEMORY_CONSUMPTION = "sm";
```
So SYSTEM is changed to be "sm" ,  change it in proxy NsonProtocol as well.